spec/models/organisation_spec.rb

```diff
@@ -0,0 +1,45 @@
+RSpec.describe Organisation do
+  describe '.find_all' do
+    context 'sorting' do
+      before do
+        create(:edition, document_type: 'organisation', title: 'Organisation C')
+        create(:edition, document_type: 'organisation', title: 'Organisation A')
+        create(:edition, document_type: 'organisation', title: 'Organisation B')
+      end
+
+      it 'returns a list of organisations' do
+        organisations = Organisation.find_all
+        expect(organisations).to all be_a(Organisation)
+      end
+
+      it 'returns a list sorted by title' do
+        organisations = Organisation.find_all
+
+        expect(organisations).to match_array([
+          have_attributes(name: 'Organisation A'),
+          have_attributes(name: 'Organisation B'),
+          have_attributes(name: 'Organisation C')
+        ])
+      end
+    end
+
+    context 'locales' do
+      before do
+        create(:edition, document_type: 'organisation', content_id: '1', title: 'English Name', locale: 'en')
+        create(:edition, document_type: 'organisation', content_id: '1', title: 'Enw Cymraeg', locale: 'cy')
```

Comment:
Shouldn't they have a different `content_id`?

Reply:
We infer `organisation id` from the `content_id`. So these tests represent when we have the same organisation (i.e. `organisation_id = 1`) with multiple locales.